include/InputManager.h

```diff
@@ -0,0 +1,24 @@
+#ifndef INPUTMANAGER_H
+#define INPUTMANAGER_H
+#include <GLFW/glfw3.h>
+
+class InputManager {
+public:
+    InputManager();
+    void Update(GLFWwindow* window);
+    bool KeyPressed(int key);
+    bool KeyReleased(int key);
+    bool MouseButtonPressed(int button);
+    bool MouseButtonReleased(int button);
+    double getMouseX();
```

Comment:
Methods should use PascalCase (every word should start with capital letter). Change it to GetMouseX and GetMouseY